### DISCUSSION TITLE
绳网跨域助手更新了还进不去

### DISCUSSION THREAD

**Discussion by Larksonguesinaspic** [share121/inter-knot#2072] [其他]
![image](https://github.com/user-attachments/assets/55ec673d-6294-458d-a067-01e42e09227e)
![image](https://github.com/user-attachments/assets/95c38fd5-610e-4fa4-beaa-23702f336288)
如图

**Comment by Genshin-open** [comment]
跟我一样哥们，绕了半天就到了这里

**Comment by share121** [comment]
![image](https://github.com/user-attachments/assets/a041f905-9ef2-42cd-9049-a9a11921693d)

可以清除缓存解决

**Reply by share121** [reply]
首先要按 F12 打开控制台

**Comment by Larksonguesinaspic** [comment]
![image](https://github.com/user-attachments/assets/6465dce6-a954-4958-ade1-868bf6fb422d)
![image](https://github.com/user-attachments/assets/cdaa8f78-0ee2-403c-bc0c-8a933d379b63)
![image](https://github.com/user-attachments/assets/151ad716-8a5d-4e80-bb12-439c6fc42d28)
还是一样啊
